src/pkg/services/m365/api/sites.go

```diff
@@ -35,6 +35,20 @@ type Sites struct {
 // methods
 // ---------------------------------------------------------------------------
 
+// GetSite returns a minimal Site with the SiteID and the WebURL
+// TODO: delete in favor of sites.GetByID()
```

Comment:
can this TODO be removed? Seems like we're calling `sites.GetByID`?

Reply:
Ah, there's corso func named `GetByID` for sites, which effectively duplicates this functionality.  I plan to dedupe those while building out the drive client.  While a bit confusing, the todo should stay for now.